lib/custom/tests/MShop/Service/Provider/Payment/DatatransQueryTest.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/**
+ * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
+ * @copyright Aimeos (aimeos.org), 2019
+ */
+namespace Aimeos\MShop\Service\Provider\Payment;
+
+class DatatransQueryTest extends \PHPUnit\Framework\TestCase
+{
+	private $object;
+	private $context;
+	private $serviceItem;
+
+	protected function setUp()
+	{
+		if ( !class_exists( 'Omnipay\Omnipay' ) ) {
+			$this->markTestSkipped( 'Omnipay library not available' );
+		}
+
+		$this->context = \TestHelper::getContext();
+
+		$serviceManager = \Aimeos\MShop\Service\Manager\Factory::create($this->context);
+		$this->serviceItem = $serviceManager->createItem();
+		$this->serviceItem->setConfig([ 'type' => 'Dummy' ]);
+		$this->serviceItem->setCode('OGONE');
+
+		$methods = [
+			'getOrder', 'getOrderBase', 'saveTransationRef', 'saveOrder', 'getProvider', 'getTransaction'
+		];
+
+		$this->object = $this->getMockBuilder(\Aimeos\MShop\Service\Provider\Payment\Datatrans::class)
+							 ->setConstructorArgs([ $this->context, $this->serviceItem ])
```

Comment:
Please only one additional indention and no mix of tabs and spaces. This applies to several lines in the test class.